tests/installation/grub_test.pm

```diff
@@ -72,11 +72,13 @@ sub run {
             type_password;    # enter PW at boot
             send_key "ret";
         }
+        workaround_type_encrypted_passphrase;
+    }
+    else {
+        # 60 due to rare slowness e.g. multipath poo#11908
+        assert_screen "grub2", 60;
+        stop_grub_timeout;
     }
-    workaround_type_encrypted_passphrase;
-    # 60 due to rare slowness e.g. multipath poo#11908
-    assert_screen "grub2", 60;
-    stop_grub_timeout;
```

Comment:
I think this function call could be obsoleted, since your module "disable_grub_timeout" is already doing this. We have to check.